python/translit/__init__.py

```diff
@@ -0,0 +1 @@
+from .engine import convert
```

Comment:
Curious, why is this required? Seems like a no-op to me.

Reply:
later you will be able to import this function by this statement:
```
from translit import convert
```

without this line you will need to do:
```
from translit.engine import convert
```